### ISSUE TITLE
assumption that every class has a superclass

### ISSUE THREAD

**Issue by pfps** [Wikidata/WikidataClassBrowser#36]
This should probably say instead that entity is not a subclass of any other class

entity (Q35120)

    thing 

something that exists
subclass of: every entity is also a(n) no value
instance of: entity is not an instance of any other class

**Comment by mkroetzsch** [comment]
I was thinking about this case, but the statement "no value" is actually stronger than not specifying a value. It says that no such class exists. Either the message should be something like "entity is a root class" or it should be flagged as an error (since entity should be a subclass at least of itself). I was undecided how to treat this (rare) case.

**Comment by pfps** [comment]
Aaah, so this is reporting on the Wikidata "no value" non-value.
Yes, there probably should be some sort of special message for this, as the current message is not particularly readable.

**Comment by mkroetzsch** [comment]
Agreed, but it is also a very rare situation. Not sure if this occurs anywhere else than on this one item.
